pulsar-broker/src/main/java/org/apache/pulsar/broker/cache/LocalZooKeeperCacheService.java

```diff
@@ -105,6 +108,38 @@ public LocalPolicies deserialize(String path, byte[] content) throws Exception {
 
                 return future;
             }
+
+            @Override
+            public CompletableFuture<Optional<Entry<LocalPolicies, Stat>>> getWithStatAsync(String path) {
```

Comment:
should we delete `getAsync(String path)` as it has duplicate logic and it is being used by bundle split only?

Reply:
Thanks, Seems a lot of other place call it indirectly, remove it will cause error of BrokerServiceTest/testCreateNamespacePolicy at least. 
Added it back in latest commit, which removed the duplicated logic.